### PR TITLE
Add j4primelte

### PR DESCRIPTION
**Pull request by GeorgeV220**
Device tree: https://github.com/GeorgeV220/android_device_samsung_j4primelte
Common tree: https://github.com/GeorgeV220/android_device_samsung_msm8917-common
Kernel sources: https://github.com/GeorgeV220/android_kernel_samsung_msm8917
Vendor: https://github.com/GeorgeV220/proprietary_vendor_samsung

### CODE REVIEW

**Comment by akhilnarang** [comment]
Link your trees not the unmodified lineage ones

**Comment by GeorgeV220** [comment]
Done. Please let me know if anything else is needed

File: res/xml/device_maintainers_fragment.xml [hunk 1]
```diff
@@ -1769,5 +1769,11 @@
             app:codename="@string/device_sunfish_codename"
             app:maintainer="@string/device_sunfish_maintainer"
             app:type="phone" />
+        <com.android.settings.rr.Preferences.DevicePreference
```
Comment (akhilnarang) [inline res/xml/device_maintainers_fragment.xml]: Keep this under samsung not under google devices

Comment (GeorgeV220) [reply res/xml/device_maintainers_fragment.xml]: Done

**Pull request merged**